terraform/stage.tfvars

```diff
@@ -0,0 +1,39 @@
+#############
```

Comment:
Should we remove the `terraform/staging.tfvars` file, or does that have to happen later?

Reply:
I want that to happen later. I'd need that file to do `terraform destroy` in that environment, so all the AWS resources will get deleted. After that I'll submit a new PR with the removal of that file